openapi.yaml

```
@@ -0,0 +1,561 @@
+openapi: 3.0.0
```

We should be able to use 3.1.0. It's worth a try I guess.

https://swagger.io/specification/

```suggestion
openapi: 3.1.0
```

---

done.